cmd/fleetctl/preview.go

```diff
@@ -27,11 +27,14 @@ const (
 func previewCommand() *cli.Command {
 	return &cli.Command{
 		Name:  "preview",
-		Usage: "Set up a preview deployment of the Fleet server",
-		UsageText: `Set up a preview deployment of the Fleet server using Docker and docker-compose. Docker tools must be available in the environment.
+		Usage: "Start a preview deployment of the Fleet server",
+		Description: `Start a preview deployment of the Fleet server using Docker and docker-compose. Docker tools must be available in the environment.
```

Comment:
i'm such a nerd but gotta admit I love reading this kind of verbiage improvement

makes sense, as it's idempotent now